letsencrypt/config.yaml

```
@@ -30,6 +31,7 @@ schema:
   domains:
     - str
   email: email
+  keytype: list(ecdsa|rsa)
```

Since this is a new option, I think it should be optional. Also, certbot only supports a single parameter as `--key-type`, so this should be a str.

Unless you want to support multiple keytypes as you alluded in the intro, but in this case you'd need to invoke certbot multiple times.

```suggestion
  keytype: str?
```

---

There's only 2 options available in certbot, so I set it to 'list(ecdsa|rsa)` just like the `challenge:` parameter did, as it seems to make it a select form input in the UI where only one option can be picked. I assumed `list(a|b)` means "either a or b" but if it's wrong then str will work. 

I agree about the optional thing but I'm gonna have to edit the run script to allow the keytype parameter to not be present.

So perhaps `keytype: list(ecdsa|rsa)?` or as you suggest `keytype: str?` depending on the first thing

---

> I assumed `list(a|b)` means "either a or b" but if it's wrong then str will work.

Yes, right, that is a list of valid values. Which is correct. Sorry about that.

Making it just optional is fine. :+1: